docs/feature_wpm.md

```diff
@@ -10,11 +10,20 @@ For split keyboards using soft serial, the computed WPM score will be available
 
 ## Configuration
 
-|Define                       |Default       | Description                                                                              |
-|-----------------------------|--------------|------------------------------------------------------------------------------------------|
-|`WPM_SMOOTHING`              |`0.0487`      | Sets the smoothing to about 40 keystrokes                                                |
-|`WPM_ESTIMATED_WORD_SIZE`    |`5`           | This is the value used when estimating average word size (for regression and normal use) |
-|`WPM_ALLOW_COUNT_REGRESSOIN` |_Not defined_ | If defined allows the WPM to be decreased when hitting Delete or Backspace               |
+| Define                       | Default       | Description                                                                              |
+|------------------------------|---------------|------------------------------------------------------------------------------------------|
+| `WPM_ESTIMATED_WORD_SIZE`    | `5`           | This is the value used when estimating average word size (for regression and normal use) |
+| `WPM_ALLOW_COUNT_REGRESSOIN` | _Not defined_ | If defined allows the WPM to be decreased when hitting Delete or Backspace               |
+| `WPM_UNFILTERED`             | _Not defined_ | If undefined (the default), WPM values will be smoothed to avoid sudden changes in value |
+| `WPM_SAMPLE_SECONDS`         | `5`           | This defines how many seconds of typing to average, when calculating WPM                 |
```

Comment:
```suggestion
| `WPM_SAMPLE_SECONDS`         | `5`           | This defines how many seconds of typing to average, when calculating WPM                 |
| `WPM_SAMPLE_PERIODS`         | `50`           | This defines how many samples to use for calculating the WPM                 |
```

Reply:
Oh geez, I actually did update the documentation with the new configurable value but seem to have missed it while copying files from one branch to the PR branch.  I've pushed an update which *does* include the updated documentation that was supposed to have been in the previous commit!